backend/src/db/factories.js

```diff
@@ -27,7 +27,7 @@ export const cleanDatabase = async (options = {}) => {
 Factory.define('category')
   .attr('id', uuid)
   .attr('icon', 'globe')
-  .attr('name', 'global-peace-nonviolence')
+  .attr('name', 'Global Peace & Nonviolence')
```

Comment:
This `name` gets translated, @mattwr18

Reply:
no, the name does not, it is the `slug` that gets translated, and because it is calling this twice, there is no key in `en.json` so it just appears as `category.name.null`. This is a separate issue that was introduced by the `rosie` refactor, but I realized it doesn't matter, cause I don't care what category the user creates the post with, so I just select the first button